### PR TITLE
dunfell: Set SRCREVs for dunfell release

### PR DESCRIPTION
**Pull request by crogers1**

### CODE REVIEW

File: recipes-openxt/surfman/surfman.inc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 PV = "0+git${SRCPV}"
 
-SRCREV = "${AUTOREV}"
+SRCREV = "46bfde5850d92399d36edb24313f2e404d53d4dd"
```
Comment (jandryuk) [inline recipes-openxt/surfman/surfman.inc:3]: Doesn't need to be done now, but this recipe should be deleted.

**Review by jandryuk — APPROVED** [review]
LGTM - I double checked all the hashes, too.

File: recipes-openxt/surfman/surfman.inc (hunk 1 above)
Comment (crogers1) [reply recipes-openxt/surfman/surfman.inc:3]: I can only think it was kept for "optional build" purposes.

**Comment by crogers1** [comment]
No functional change, merge whenever.

**Pull request merged**